Mobile Test Runner for Jama/Mobile Test Runner for Jama/UserModel.swift

```diff
@@ -14,6 +14,19 @@ class UserModel {
     var firstName: String = ""
     var lastName: String = ""
     var email: String = ""
-    var id: Int = 0
+    var id: Int = -1
     var username: String = ""
+
+    func extractUser(fromData: [String : AnyObject]) {
+        if fromData["firstName"] == nil || fromData["lastName"] == nil || fromData["email"] == nil || fromData["username"] == nil || fromData["id"] == nil {
```

Comment:
Ok. If any of them are not required, it can be fixed later.  Let me know if there's anything you'd like to ask Jama, or you can always contact Iman via slack.